pkg/cri/server/image_pull.go

```diff
@@ -126,10 +128,17 @@ func (c *criService) PullImage(ctx context.Context, r *runtime.PullImageRequest)
 		}
 	)
 
+	defer pcancel()
+	snapshotter, err := c.snapshotterFromPodSandboxConfig(ctx, ref, r.SandboxConfig)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to get snapshotter for %q", ref)
+	}
+	log.G(ctx).Debugf("PullImage %s with snapshotter %s", ref, snapshotter)
```

Comment:
see above Debugf for pattern "PullImage ref: %q with ..."

Reply:
Updated